### PR TITLE
add an example of adding an A object to Search

### PR DESCRIPTION
**Pull request by benregn**
Solves #168

### CODE REVIEW

File: docs/search_dsl.rst [hunk 1]
```diff
@@ -218,6 +218,24 @@ acts as a top-level aggregation:
 .. code:: python
 
     s = Search()
+    a = A('terms', field='category')
+    s.aggs.metric('category_terms', a)
```
Comment (HonzaKral) [inline docs/search_dsl.rst]: If you are adding a `terms` aggregation, it should be added as `bucket`, not metric.

**Comment by HonzaKral** [comment]
Thank you so much for the PR! I added a tiny comment on one line, and I would also ask you to sign our CLA (https://www.elastic.co/contributor-agreement/), if you haven't already, so I can merge it.

**Comment by benregn** [comment]
@HonzaKral changed ``metric`` to ``bucket`` :)

**Comment by HonzaKral** [comment]
Thanks!

**Pull request merged**